src/main/java/org/broadinstitute/hellbender/engine/TargetWalker.java

```diff
@@ -24,11 +31,5 @@ protected boolean isAcceptableFeatureType(final Class<? extends Feature> feature
     }
 
     @Override
-    public File getDrivingFeatureFile() {
-        final File targetsFile = targetArguments.getTargetsFile();
-        if (targetsFile == null) {
-            throw new UserException.BadArgumentValue(TargetArgumentCollection.TARGET_FILE_FULL_NAME, "you must specify a target file");
-        }
-        return targetsFile;
-    }
+    public File getDrivingFeatureFile() { return targetsFile; }
```

Comment:
This walker used to have a `TargetArgumentCollection`, which has an optional targets file along with other arguments.  Since 1) these other arguments were not used and 2) the targets file is never optional for this walker, I decided to use the straightforward approach of simply having a non-optional command line argument for the targets file.